Changelog.md

```diff
@@ -2,7 +2,7 @@
 
 This file only reflects the changes that are made in this image. Please refer to the upstream GitLab [CHANGELOG](https://gitlab.com/gitlab-org/gitlab-foss/blob/master/CHANGELOG.md) for the list of changes in GitLab.
 
-**16.10.6**
+**16.10.7**
 
 - gitlab: upgrade CE to v16.10.6
 - gitaly: upgrade to v16.10.6
```

Comment:
Good point! And the each update entries may need to be fixed as well. Could you please fix it?

```suggestion
**16.10.7**

- gitlab: upgrade CE to v16.10.7
- gitaly: upgrade to v16.10.7
```